### PR TITLE
Create db-reconstruction.md

### PR DESCRIPTION
**Pull request by PxDave**
Document recovery procedure for Cockroach database for existing PVC

### CODE REVIEW

File: docs/db-reconstruction.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+b. attached volumes stay attached - existing container mounts will not be disturbed.
+c. new volumes will fail to bind and new attachments will fail.
+
+
+2. Destroy and eradicate all backend Cockroach database volumes.  Optionally - these DB volumes can be renamed on the FlashArray ( PSO will ignore DB volumes with "-u" appended to the volume name.  This labels them as "unusable" and the CR DB will not attempt to initialize or used them. )  FlashBlade will not allow the volume to be renamed - so in the case of CR DB volumes residing on FlashBlade, the protocol must be disabled so the FlashBlade volume can be deleted and eradicated.  In environments where both FlashArray and FlashBlade are used together, the Cockroach database copies are distributed over those backend appliances for redundancy.
+
+3  Reinstall PSO: 
+
+   This would be a good time to review the [installation instructions](https://github.com/purestorage/pso-csi/blob/master/pure-pso/README.md)
+   Be sure to pull a fresh, new copy of the values.yaml as mentioned.
```
Comment (dsupure) [inline docs/db-reconstruction.md:23]: Why we need a new copy of `values.yaml`? Shouldn't we use the same one? Btw, we should also emphasize the `clusterID` remains the same.

File: docs/db-reconstruction.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+3  Reinstall PSO: 
+
+   This would be a good time to review the [installation instructions](https://github.com/purestorage/pso-csi/blob/master/pure-pso/README.md)
+   Be sure to pull a fresh, new copy of the values.yaml as mentioned. 
+
+At this point, PSO will be functioning, but with an empty database. New provisions will work and those volumes will be fine, but old volume will not work as they are not in the database.
+
+4. On the first run of the reconstructor - there will likely be an error about missing CRDs, there are three snapshot CRD manifests to install These should probably be installed after the helm install of PSO has completed.
```
Comment (dsupure) [inline docs/db-reconstruction.md:27]: Can we change our description or a note saying if you use volumes snapshots, please install the three snapshot BETA CRDs before run the db reconstruction tool?

File: docs/db-reconstruction.md (hunk 1 above)
Comment (PxDave) [reply docs/db-reconstruction.md:23]: will add emphasis for clusterID.  Fresh values.yaml is to ensure app container versions and pull policy are correct or at least prompt the user to review for any differences.

File: docs/db-reconstruction.md (hunk 2 above)
Comment (PxDave) [reply docs/db-reconstruction.md:27]: From my field experience, the tool requires the BETA CRDs even if the user never created or used snapshots.  I will change the wording as suggested.

**Comment by PxDave** [comment]
Incorporated feedback and reworded, make format consistent.

**Review by dsupure — APPROVED** [review]
LGTM. Thanks for prepare the db reconstruction guide. @PxDave

**Review by haibinxie — APPROVED** [review]

**Comment by PxDave** [comment]
I do not have write access, so cannot merge myself

**Pull request merged**